src/components/ReportActionItem/MoneyRequestPreview.js

```diff
@@ -159,6 +164,22 @@ function MoneyRequestPreview(props) {
         description = props.transaction.merchant;
     }
 
+    // Prevents large amounts from being cut off on small screen widths.
+    const getFontSizeAndLineHeightToSubtract = () => {
+        let toSubtract = 0;
+        if (isSmallScreenWidth) {
+            const widthDifference = variables.mobileResponsiveWidthBreakpoint - windowWidth;
+            if (widthDifference > 450) toSubtract = 9;
+            else if (widthDifference > 400) toSubtract = 6;
+            else if (widthDifference > 350) toSubtract = 2;
+        }
+
+        // requestAmount also includes digits after ".", so "1,000,000.00" qualifies.
+        if (requestAmount >= 100000000) toSubtract += 2;
+
+        return toSubtract;
+    };
```

Comment:
Possibly better suited to a helper function, as we can separate this from the component and it could be useful elsewhere in the future.

Reply:
@Julesssss Is there currently a fitting file/folder in which to move this? Thanks!

Reply:
Maybe `src/styles/StyleUtils.js`

Reply:
@Julesssss moved it. Does it look good?